### PR TITLE
Example: compare

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: Tasks/compare.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Compare two dictionaries
+
+let compare=(
```
Comment (Kate732) [inline Tasks/compare.js:3]: Don't have to use let, but const

File: Tasks/compare.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  (first_values,...parameters_LIST) => {
+   const second_values = parameters_LIST[0]
```
Comment (Kate732) [inline Tasks/compare.js:5]: We can use just obj, don't need to create list

File: Tasks/compare.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+   let a = Object.keys(first_values)
+   let b = Object.keys(second_values)
+   if (a.join('-') !== b.join('-')) return false;
+   let e = true;
+   for (c of a) {
```
Comment (Kate732) [inline Tasks/compare.js:10]: We can rename smth

**Comment by Kate732** [comment]
// my version of this code js

const compare = (firstArr, arrToCompare) => {
    let keysFromFirst = Object.keys(firstArr);
    let keysFromSecond = Object.keys(arrToCompare);
    if (keysFromFirst.join('-') !== keysFromSecond.join('-')) return false;
    let res = true;
    for (key of keysFromFirst) {
        if (firstArr[key] === arrToCompare[key]) res = res && true;
        else res = res && false;
    }
    return res;
}

const result = compare({ a: 1, c: 'hello' }, { a: 1, c: 'hello' });
console.log(result);

**Comment by Kate732** [comment]
```
# my version of code py

def compare(firstArr, arrToCompare):
    keysFromFirst = firstArr.keys()
    keysFromSecond = arrToCompare.keys()
    if keysFromFirst != keysFromSecond:
         return False
    res = True
    for key in keysFromFirst:
        if firstArr[key] == arrToCompare[key]: res = res & True
        else: res = res & False
    return res


result = compare({ "a": 1, "c": 'hello' }, { "a": 1, "c": 'hello' })
print(result)
 ```

**Comment by tshemsedinov** [comment]
@Kate732 just a hint, after triple-back-tick add language name, like this: &#96;&#96;&#96;js or &#96;&#96;&#96;py
I have corrected this in your post )

**Comment by tshemsedinov** [comment]
```js
if (firstArr[key] === arrToCompare[key]) res = res && true;
else res = res && false;
```

**Pull request closed without merging**